src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -15,7 +15,7 @@
 /**
  * Represents the file used to store address book data.
  */
-public class StorageFile {
+public class StorageFile extends Storage{
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! One more more of these problems detected:
1. missing space after `Storage`